sources/HUBComponentImageDataBuilderImplementation.h

```diff
@@ -31,6 +31,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Concrete implementation of the `HUBComponentImageDataBuilder` API
 @interface HUBComponentImageDataBuilderImplementation : NSObject <HUBComponentImageDataBuilder, NSCopying>
 
+/// Any specific bundle that the builder should use to load local images (defaults to the main bundle)
+@property (nonatomic, weak, nullable) NSBundle *bundle;
```

Comment:
Well that means the user of the framework will have to retain the `NSBundle` object for the framework. Which makes it a bit more hairy and tricky to use. Since there’s no guarantee that the system will retain it for you.